### PR TITLE
[Spanish] docs(es): add examples book 

### PR DESCRIPTION
**Pull request by Kronhyx**
- [x] customize-loading-indicator.md
- [x]  customize-nuxt-loading.md
- [x] custom-loading-component.md
- [x] data-fetching-async-data.md
- [x] data-fetching-fetch-hook.md
- [x] hello-world.md
- [x] helpers.md
- [x] layouts.md
- [x] lazy-loading-components.md
- [x] middleware-anonymous.md
- [x] middleware-named.md
- [x] middleware-router.md
- [x] modules-axios.md
- [x] modules-internal.md
- [x] plugins-client-only.md
- [x] plugins-custom.md
- [x] plugins-external-packages.md
- [x] plugins-vue.md
- [x] pre-processors.md
- [x] routing-active-links-classes.md
- [x] routing-dynamic-pages.md
- [x] routing-nested-pages.md
- [x] seo-html-head.md
- [x] seo-twitter-og.md
- [x] transitions.md
- [x] vuex-store.md
- [x] webpack-assets.md

### CODE REVIEW

**Comment by Kronhyx** [comment]
@debs-obrien , can u add the spanish tag?

**Comment by debs-obrien** [comment]
hey @Kronhyx sure, can you also change the title to say docs(es) instead of feat(i18n)

**Comment by miguelalcala-github** [comment]
Hi @Kronhyx !! I was looking to contribute in docs(es) translations and its different sections like you are doing with examples and I'd like to help here if you don't mind

**Comment by Kronhyx** [comment]
yes, of course @miguelalcala-github, divide the remaining translations in half, and send me what they are to remove them from here

**Comment by miguelalcala-github** [comment]
I was thinking more in make pr in yours, or if there is a better way, to keep a compact way to merge all examples.

**Comment by Kronhyx** [comment]
> I was thinking more in make pr in yours, or if there is a better way, to keep a compact way to merge all examples.

@miguelalcala-github, ahh.. me parece bien igual... como lo haríamos? debería darte permisos en mi repositorio? o existe otra manera?

**Comment by miguelalcala-github** [comment]
Vale @Kronhyx ya he hecho una prueba y si, necesitaria que me dieras permiso para poder hacer el push.
No te preocupes que no hay prisa tampoco. Buen dia!

```nuxtjs.org % git push kronhyx HEAD:i18n/es_examples
remote: Permission to Kronhyx/nuxtjs.org.git denied to miguelalcala-github.
fatal: unable to access 'https://github.com/Kronhyx/nuxtjs.org.git/': The requested URL returned error: 403

**Comment by Kronhyx** [comment]
@miguelalcala-github 

> ```
> remote: Permission to Kronhyx/nuxtjs.org.git denied to miguelalcala-github.
> fatal: unable to access 'https://github.com/Kronhyx/nuxtjs.org.git/': The requested URL returned error: 403
> ```

image

**Comment by miguelalcala-github** [comment]
Ok! Ahi va!

docs(es): translate transitions.md

https://github.com/Kronhyx/nuxtjs.org/commit/b9985fbb8ed49e1ef90b3051b64b8714fc1e4c6b

Perhaps it would be good to have certain terms in common: for example 'features' that can be funciones / características, I had doubts between them but in the Spanish guide it is already as ¨características¨ so I thought about translating as this one.

**Comment by Kronhyx** [comment]
> Ok! Ahi va!
> 
> docs(es): translate transitions.md
> 
> [Kronhyx@b9985fb](https://github.com/Kronhyx/nuxtjs.org/commit/b9985fbb8ed49e1ef90b3051b64b8714fc1e4c6b)
> 
> Perhaps it would be good to have certain terms in common: for example 'features' that can be funciones / características, I had doubts between them but in the Spanish guide it is already as ¨características¨ so I thought about translating as this one.

Yes, I fully agree @miguelalcala-github , just remember to check the box of the file you plan to translate, to avoid translating the same file

- [x] **transitions.md**

**Comment by miguelalcala-github** [comment]
Ok... and sorry for not knowing, but how can I check the box? I can't edit your first comment and I don't think you mean to write it in new commets each time.
Just in case...  I'm going to work these:
- [x] data-fetching-async-data.md
- [x] data-fetching-fetch-hook.md

**Comment by miguelalcala-github** [comment]
So next:
- [x] middleware-anonymous.md
- [x] middleware-named.md
- [x] middleware-router.md

**Comment by debs-obrien** [comment]
Keep up the great work guys. 💪🏻@alvarosaburido or @Dawntraoz would you mind giving these the once over to make sure things that shouldn't be translated don't get translated or so that chapter names remain same across examples and Spanish docs. You both have done such an excellent job in reviewing the docs and setting standards so would be great if you could advise Randy and Miguel as right now we have different translations for named middleware. Middleware nombrado in docs and middleware declarado in examples among others. These should be consistent. It might be an idea @Kronhyx and @miguelalcala-github to make sure you open the docs of the chapter the example refers to to also see what words have been used as these pages have gone through revision and should be ok however by all means if you something that seems wrong just let us know. Thanks everyone 😃 and keep up the great work

**Comment by miguelalcala-github** [comment]
@debs-obrien it's my fault, I revised every chapter in the (es) guide to be sure to use the same words but also thought that better to use words that are not so 'literal', in this case 'middleware declarado' comes from 'function declaration' that its indeed a named function versus an anonimous function or in our case, 'anonymous middleware'. 
I recognise my error, I was thinking on propose this change after set it up on the example but I supose is better if I had proposed it before set it.
@alvarosaburido  or @Dawntraoz:  May I change 'middleware declarado'  and leave it as 'middleware nombrado' or we can see if it is more suitable to change the guide to 'middleware declarado'? Sorry, I always studied development in english and even I have looked for the best and meaningful term for this translation I haven't found it but 'nombrado' doesn't seem to really fit, for me, and again I'm sorry for take this privileges because its my perspective and surely I am wrong.

**Comment by debs-obrien** [comment]
no problem @miguelalcala-github and great point regarding naming. But yes always best to propose in pr if adding a naming structure that is different to the docs or at least to add a comment saying why you have changed it and make changes in other pages too. Then other people can help decide which sounds better. As a non native speaker I cannot tell you which is best so will leave that for the native Spanish speakers to advise on.

Other things I noticed are - Obtención de datos in your pr where book chapter in Spanish is Obtener datos. Just small details but best to keep it consistent. Which on is better? I have no idea but they should be the same so just use the chat here to ask questions if you think something should be changed or looks wrong in your opinion. @alvaroOnCode and @Dawntraoz and even @Kronhyx can help you more on these naming conventions than me. So don't be afraid to use the comments to ask questions and get advice. We are all here to help and will try to help as much as we can.

Thanks and keep up the great work.

**Comment by debs-obrien** [comment]
@Kronhyx: new example added today - https://nuxtjs.org/examples/seo-twitter-og

**Comment by alvarosaburido** [comment]
I'm finally free these first weeks of January, what I can help out with? @debs-obrien @Kronhyx

**Review by alvarosaburido — CHANGES_REQUESTED** [review]
Hey guys @Kronhyx @miguelalcala-github I reviewed all the changes and in general, is all good some typos and I would use only one translation of Features book,  in some translations is `Libro de Características` and in the previous document is called `Libro Funciones`.

In my opinion, the correct translation would be `libro de Características`. If you want to highlight it with bold text the better.

If you give me access to your repo I can help you out with the rest of the pages if you like

File: content/es/examples/seo-twitter-og.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+<base-alert type="next">
+
+Aprende más sobre las opciones disponibles para `head`, en la [documentación de vue-meta](https://vue-meta.nuxtjs.org/api/#metainfo-properties).
+
+</base-alert>
+
+<base-alert type="next">
+
+Aprende más sobre las metaetiquetas en el libro Funciones en [Metaetiquetas y SEO](/docs/2.x/features/meta-tags-seo) chapter.
```
[the remark this answers is not in the document]
Comment (Kronhyx) [reply content/es/examples/seo-twitter-og.md:25]:
```suggestion
Aprende más sobre las metaetiquetas en el capítulo de [Metaetiquetas y SEO](/docs/2.x/features/meta-tags-seo) en el libro de Características .
```

**Review by alvarosaburido — DISMISSED** [review]

**Comment by debs-obrien** [comment]
Great work guys. Thanks @alvarosaburido for taking the time to review and @Kronhyx for applying the suggestions. Should I merge these examples even though there are some that will be in English and some in Spanish or is anyone still actively working on them?

**Comment by miguelalcala-github** [comment]
Sorry for my absence. 
I know there is work to do and left it undone.
First I've done is fixed those naming conventions [ ~~Obtención de datos~~ (Obtener datos) and ~~declarado~~(nombrado)] https://github.com/Kronhyx/nuxtjs.org/commit/6a363f49b34beed7ed0e998a215157b6d3e62887
I'm going to continue with:
- [ ] layouts.md
- [ ] lazy-loading-components.md

And thank you for your reviews and patience!

**Comment by miguelalcala-github** [comment]
Hi again!!
Pending;
- [ ] modules-axios.md
- [ ] modules-internal.md
- [ ] plugins-client-only.md
- [ ] plugins-custom.md
- [ ] plugins-external-packages.md
- [ ] plugins-vue.md

**Comment by miguelalcala-github** [comment]
Hello there!!
Work in progress:

**Comment by miguelalcala-github** [comment]
Hello there!
Work in progress:
- [ ] routing-dynamic-pages.md
- [ ] routing-nested-pages.md

**Comment by debs-obrien** [comment]
Nice work @miguelalcala-github 💪

**Comment by miguelalcala-github** [comment]
Hello!!
Thank you!!
Work in progress:
- [ ] pre-processors.md
- [ ] seo-html-head.md

**Comment by miguelalcala-github** [comment]
Friyay!!!
Work in progress:
- [ ] vuex-store.md
- [ ] webpack-assets.md

**Comment by debs-obrien** [comment]
@alvarosaburido @Kronhyx @Dawntraoz if any of you have time can you give these the once over before we merge.

Great work @miguelalcala-github and thanks for your nice contribution 💪

File: content/es/examples/pre-processors.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+---
+title: Preprocesadores
+description: Configura tu aplicación para hacer uso de pug y sass con recursos de estilo y así añadir fácilmente variables a todos los componentes.
```
Comment (alvarosaburido) [inline content/es/examples/pre-processors.md:3]:
```suggestion
description: Configura tu aplicación para hacer uso de Pug y Sass con recursos de estilo y así añadir fácilmente variables a todos los componentes.

```

File: content/es/examples/pre-processors.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+position: 2
+category: assetManagement
+csb_link: https://codesandbox.io/embed/github/nuxt-academy/examples/tree/master/asset-management/pre-processors?fontsize=14&hidenavigation=1&theme=dark&view=editor
+---
+
+<example-intro></example-intro>
+
+`pages/index.vue` hace uso de pug como lenguaje de la plantilla y sass para el estilo.
```
Comment (alvarosaburido) [inline content/es/examples/pre-processors.md:11]:
```suggestion
`pages/index.vue` hace uso de Pug como lenguaje de la plantilla y Sass para el estilo.
```

**Review by alvarosaburido — CHANGES_REQUESTED** [review]
Hi @miguelalcala-github excellent work, just minor details like dots `.` at the end of phrases and some tech names that should be capitalized. The rest is all ok

As soon they are corrected, you have my approval.

Great job!

File: content/es/examples/middleware-named.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
+---
+title: Middleware nombrado
```
[the remark this answers is not in the document]
Comment (Dawntraoz) [reply content/es/examples/middleware-named.md:2]: Es una traducción difícil 🤦
Pero yo diría que Middleware con nombre/Middleware anónimo se entiende mejor, no?

[the remark this answers is not in the document]
Comment (miguelalcala-github) [reply content/es/examples/middleware-named.md:2]: Hi! I'm working on this changes, thank you @alvarosaburido.
About Middleware, in (es) guide is set like that, "nombrado". 
https://github.com/nuxt/nuxtjs.org/pull/1012#issuecomment-735408624 message where I explain that maybe it's better "declarado" but only to take an option.

[the remark this answers is not in the document]
Comment (alvarosaburido) [reply content/es/examples/middleware-named.md:2]: Declarado suena mejor la verdad 🤔

**Comment by debs-obrien** [comment]
perfecto. Gracias a todos. 💚

**Pull request merged**